src/modules/75-cd/pages/webhooks/utils.tsx

```diff
@@ -23,17 +23,16 @@ export interface NewWebhookModalProps {
   entityScope?: Scope
   closeModal: () => void
 }
+export function processFolderPaths(folderPaths: string[]): { id: string; value: string }[] {
+  return [...folderPaths].map(path => ({ id: uuid('', nameSpace()), value: path }))
```

Comment:
NIT: This copy is not needed as map in immutable. 
```suggestion
  return folderPaths.map(path => ({ id: uuid('', nameSpace()), value: path }))
```

Reply:
done